assignments/atea/01/Atea.Scraper/Atea.Scraper/Services/StorageService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Azure.Data.Tables;
+using Azure.Storage.Blobs;
+
+namespace Atea.Scraper.Services;
+
+public class StorageService : IStorageService
+{
+    private readonly TableServiceClient _tableServiceClient;
+    private readonly BlobServiceClient _blobServiceClient;
+
+    public StorageService()
+    {
+        var connectionString = "UseDevelopmentStorage=true";
+        _tableServiceClient = new TableServiceClient(connectionString);
+        _blobServiceClient = new BlobServiceClient(connectionString);
+    }
+
+    public async Task<TableClient> GetOrCreateTableClientAsync(string tableName)
```

Comment:
```suggestion
    public async Task<TableClient> GetTableClientAsync(string tableName)
```

nav vajadzības likt create. :) 
tā ir implementācijs detaļa kas gala lietotājam nav obligāti jāzin. :)